mobile/lib/features/trade/channel_configuration.dart

```diff
@@ -141,8 +141,12 @@ class _ChannelConfiguration extends State<ChannelConfiguration> {
 
   @override
   Widget build(BuildContext context) {
+    // We add a buffer because the `fundingTxFee` is just an estimate. This
+    // estimate will undershoot if we end up using more inputs or change
+    // outputs.
+    final Amount fundingTxFeeWithBuffer = Amount(fundingTxFee.sats * 2);
     final maxUsableOnChainBalance =
```

Comment:
Could we end up with dust value because of this ?

Let's assume the estimated fee is 1000 sats. If we overestimate * 2, we get 2000 sats estimate. We end up needing 1200 sats and 800 will remain.

Reply:
It's a good question. I think so, although it should be the responsibility of the coin selection algorithm to ensure that no dust change output is generated.